app/assets/javascripts/Components/Modals/filter_modal.jsx

```diff
@@ -0,0 +1,325 @@
+                      this.props.mutateFilterData({
+                        ...this.props.filterData,
+                        markingState: selection,
+                      });
+                    }}
+                  />
+                </div>
+              </div>
+              <div className={"modal-container"}>
+                <div className={"filter"}>
+                  <p>{I18n.t("activerecord.models.tag.other")}</p>
+                  {this.renderTagsDropdown()}
+                </div>
+                <div className={"filter"} data-testid={"section"}>
+                  <p>{I18n.t("activerecord.models.section.one")}</p>
+                  <SingleSelectDropDown
+                    options={this.props.sections}
+                    selected={this.props.filterData.section}
+                    onSelect={selection => {
+                      this.props.mutateFilterData({
+                        ...this.props.filterData,
+                        section: selection,
+                      });
+                    }}
+                    defaultValue={""}
+                  />
+                </div>
+              </div>
+              <div className={"modal-container"}>
+                {this.renderTasDropdown()}
+                <div className={"annotation-input"}>
+                  <p>{I18n.t("activerecord.models.annotation.one")}</p>
+                  <input
+                    type={"text"}
+                    value={this.props.filterData.annotationText}
+                    onChange={e =>
+                      this.props.mutateFilterData({
+                        ...this.props.filterData,
+                        annotationText: e.target.value,
+                      })
+                    }
+                    placeholder={I18n.t("results.filters.text_box_placeholder")}
+                  />
+                </div>
+              </div>
+
+              <div className={"modal-container"}>
+                {this.rangeFilter(
+                  this.props.filterData.totalMarkRange.min,
+                  this.props.filterData.totalMarkRange.max,
+                  I18n.t("results.filters.total_mark"),
+                  this.onTotalMarkMinChange,
+                  this.onTotalMarkMaxChange
+                )}
+                {this.rangeFilter(
+                  this.props.filterData.totalExtraMarkRange.min,
+                  this.props.filterData.totalExtraMarkRange.max,
+                  I18n.t("results.filters.total_extra_mark"),
+                  this.onTotalExtraMarkMinChange,
+                  this.onTotalExtraMarkMaxChange
+                )}
+              </div>
+            </div>
+          </div>
+          <div className={"modal-footer"}>
+            <section className={"modal-container dialog-actions"}>
+              <input type="reset" value={I18n.t("clear_all")} onClick={this.onClearFilters} />
+              <input type="submit" value={I18n.t("close")} />
```

Comment:
Change this to a `button` element, since it's not triggering a form submission. Note that this button can actually be disabled/not work if the number ranges are invalid